pages/index.js

```diff
@@ -1,13 +1,13 @@
 import MatchList from "../components/MatchList";
-import AppHeader from "../components/AppHeader/AppHeader";
-import AppFooter from "../components/AppFooter/AppFooter";
+import AppHeader from "../components/AppHeader/index.js";
```

Comment:
See above comment ;)

Reply:
Oh understood, since it is an default export, right? 
I will adjust this in my next US :-)

Reply:
Done :-)